src/Tribe/Assets.php

```diff
@@ -6,8 +6,10 @@ class Tribe__Tickets__Assets {
 	 * @since 4.6
 	 */
 	public function enqueue_scripts() {
+		$tickets = Tribe__Tickets__Main::instance();
+
 		tribe_assets(
-			Tribe__Tickets__Main::instance(),
+			$tickets,
```

Comment:
Advice: I like to be bleedingly obvious, so I'd prefer `$tickets_main` (or something similar) for this variable - mainly because `$tickets` is _so_ common and easy to misuse (accidentally) in this plugin?